### ISSUE TITLE
Provide cmd line argument to seed training

### ISSUE THREAD

**Issue by demongolem** [arthurflor23/handwritten-text-recognition#16]
When I train the same combo of architecture and source a number of times, I get drastically different results as far as epochs (and hence training time) as well as validation loss, but I need to do some benchmark testing.  It would be helpful to seed the training process so that I can reproduce my results if necessary.

**Comment by uchreet-godara** [comment]
Does changing dimension for training  will work. I mean I want to train this model on (1024,512 ) instead of given(1024,128) shape and use those model weights for further  training on my data. Does that work?

**Comment by arthurflor23** [comment]
Just set the seed with numpy/tensorflow in the code
Numpy: https://docs.scipy.org/doc/numpy-1.15.0/reference/generated/numpy.random.seed.html
Tensorflow: https://www.tensorflow.org/api_docs/python/tf/random/set_seed
